Core/HandlerMgr.py

```diff
@@ -55,17 +58,19 @@ def __calculateRoutes(self):
     Load all handlers and generate the routes
     """
     ol = ObjectLoader()
-    origin = "WebApp.handler"
-    result = ol.getObjects(origin, parentClass=WebHandler, recurse=True)
-    if not result['OK']:
-      return result
-    self.__handlers = result['Value']
+    self.__handlers = collections.OrderedDict()
+    for origin in self.__sysService:
+      result = ol.getObjects(origin, parentClass=WebHandler, recurse=True)
+      if not result['OK']:
+        return result
+      self.__handlers = collections.OrderedDict(list(self.__handlers.items()) + list(result['Value'].items()))
```

Comment:
You recreating the self.__handlers few times. Could you please add the new handlers to the orderedDict instead of recreate len(self.__sysService) time?

Reply:
fixed